### PR TITLE
Add openSUSE Leap 42.1 for Liberty and Master

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/jenkins/jobs-obs/openstack-cleanvm.yaml [hunk 1]
```diff
@@ -31,8 +32,8 @@
         && ( openstack_project=="Icehouse" && (image=="openSUSE-13.1-64-minimal" || image=="SLE_11_SP3-64-minimal"))
         || ( openstack_project=="Juno" && (image=="openSUSE-13.2-64-minimal" || image=="SLE_11_SP3-64-minimal"))
         || ( openstack_project=="Kilo" && (image=="openSUSE-13.2-64-minimal" || image=="SLE_12-64-minimal"))
-        || ( openstack_project=="Liberty" && (image=="SLE_12_SP1-64-minimal" || image=="SLE_12-64-minimal" || image=="openSUSE-13.2-64-minimal"))
-        || ( openstack_project=="Master" && (image=="openSUSE-13.2-64-minimal" || image=="SLE_12-64-minimal"))
+        || ( openstack_project=="Liberty" && (image=="SLE_12_SP1-64-minimal" || image=="SLE_12-64-minimal" || image=="Leap-42.1-64-minimal"))
+        || ( openstack_project=="Master" && (image=="Leap-42.1-64-minimal" || image=="SLE_12-64-minimal"))
```
Comment (aplanas) [inline scripts/jenkins/jobs-obs/openstack-cleanvm.yaml]: Do we want SP1 in Master?

File: scripts/jenkins/jobs-obs/openstack-cleanvm.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@
             - SLE_12-64-minimal
             - SLE_12_SP1-64-minimal
             - openSUSE-13.2-64-minimal
+            - Leap-42.1-64-minimal
```
Comment (aplanas) [inline scripts/jenkins/jobs-obs/openstack-cleanvm.yaml]: The real name is openSUSE Leap 42.1, I would like to maintain the openSUSE prefix here

**Comment by bmwiedemann** [comment]
:+1:

**Comment by aplanas** [comment]
+1

**Comment by jdsn** [comment]
:+1:

**Pull request merged**